### ISSUE TITLE
⌨ Unclear what is selected / unselected in community details

### ISSUE THREAD

**Issue by BrookeCameron** [GCTC-NTGC/gc-digital-talent#13393]
### ⌨ Accessibility and Expected Behaviour

There is no way for users using screen readers with VoiceOver to tell what is selected or not. 

### 📋 Steps to Reproduce

1. Open community details 
2. tab through items 

### 🕵️ Details

This issue can be reproduced with VoiceOver on Chrome as well. 

### What operating systems are you encountering the problem on?

macOS

### What browsers are you encountering the problem on?

Firefox

### What assistive technologies are you encountering the problem on?

VoiceOver

### 📸 Screenshot

https://github.com/user-attachments/assets/edf83771-e059-495e-8cf6-ed9ae0282c1f

**Comment by BrookeCameron** [comment]
Question for @GCTC-NTGC/designers for this instance (Functional communities details) can we show only selected information?

**Comment by petertgiles** [comment]
We can add accessible labels to these, too.  We would need some help with what the labels should be.

Financial Management Advisory Services (is not interesting for job and training opportunities)
Financial Policy (is interesting for job and training opportunities)

**Comment by mnigh** [comment]
Related https://github.com/GCTC-NTGC/gc-digital-talent/issues/13343.

**Comment by BrookeCameron** [comment]
I still feel like it makes the most sense for the selected items to be group together on the top and then have a not selected group below.
